gdg_events/android_study_jam.md

```diff
@@ -3,4 +3,39 @@
 
 ![](http://developerstudyjams.com/images/masthead.png)
 
+- Google Developers Study Jams is a free series of global, community-run, in-person study groups. 
+- The first Study Jams series was presented in partnership with Udacity and will follow their Android Fundamentals curriculum. 
+- Google Developer Group facilitators certified in the course lead the Study Jams that meet once weekly for 8 - 9 weeks.
+- Study Jams follow the free version of the course on udacity.
+- Promoted using hashtag `#devstudyjam`
 
+
+
+#####WHAT IS GOOGLE DEVELOPERS STUDY JAMS?
+Google Developers Study Jams is an 8-week series of in-person, community-run study groups to support students taking online Udacity courses on Google developer technologies. These events are run by local Google Developer Groups or other developer meetups all over the world. This season, we are running Study Jams around the Udacity [Developing Android Apps: Android Fundamentals](https://www.udacity.com/course/developing-android-apps--ud853) course.
+
+
+
+#####WHAT HAPPENS AT A STUDY JAM?
+Study Jams are an opportunity for students to receive free, in-person guidance and tutoring on the Udacity course in a fun classroom environment. Study Jam sessions are designed around the Udacity course structure, with a weekly session following each lesson (6 lessons), plus two additional weeks for completing and sharing course final projects. Students are expected to come to the Study Jam sessions having completed that week’s Udacity lesson online, so that the Study Jam time may be used to review key concepts, address questions, and work on individual projects. The live community atmosphere and tutoring is designed to help students achieve deeper learning around the concepts taught in the course.
+
+
+
+#####WHAT ARE THE REQUIREMENTS TO JOIN A STUDY JAM?
+All are welcome to join a Study Jam, though students should first be sure to meet the course prerequisites and requirements: [Developing Android Apps: Android Fundamentals](https://www.udacity.com/course/ud853) is intended for students with at least 3 years of programming experience in Java or another object­-oriented programming language. You will be expected to download Android Studio in order to follow along with the instructors throughout the course. You should also be comfortable working with code on Github. Access to an Android device is helpful, but not required, to complete the final project. Please see [Technology Requirements](https://www.udacity.com/tech-requirements) for using Udacity.
```

Comment:
Again, Study Jam vs Android Study Jam.

You are expected to bring a laptop or use a computer on-site.